src/rate.rs

```diff
@@ -24,6 +24,31 @@ pub const FRAME_SUBTYPE_B1: usize = 3;
 // The scale of AV1 quantizer tables (relative to the pixel domain), i.e., Q3.
 const QSCALE: i32 = 3;
 
+// The base quantizer for a frame is adjusted based on the frame type using the
+//  formula (log_qp*mqp + dqp), where log_qp is the base-2 logarithm of the
+//  "linear" quantizer (the actual factor by which coefficients are divided).
+// Because log_qp has an implicit offset built in based on the scale of the
+//  coefficients (which depends on the pixel bit depth and the transform
+//  scale), we normalize the quantizer to the equivalent for 8-bit pixels with
+//  orthonormal transforms for the purposes of rate modeling.
+const MQP_Q12: &[i32; FRAME_NSUBTYPES] = &[
+  // TODO: Use a const function once f64 operations in const functions are
+  //  stable.
+  (1.0 * (1 << 12) as f64) as i32,
+  (1.0 * (1 << 12) as f64) as i32,
+  (1.0 * (1 << 12) as f64) as i32,
+  (1.0 * (1 << 12) as f64) as i32
```

Comment:
Are these all intended to be the same?

Reply:
Yes, these are intended to intially be the same as the current boosting does not scale with the quantizer. In the near future, we will tune the values in `MQP_Q12` and `DQP_Q57` but for now we just need to capture the current boosting in linear form.